ps_linklist.php

```diff
@@ -136,34 +132,32 @@ private function installFixtures()
     {
         $installed = true;
         $errors = $this->getRepository()->createTables();
-        if (!empty($errors)) {
-            $this->addModuleErrors($errors);
+        if (false === $errors || (is_array($errors) && !empty($errors))) {
```

Comment:
```suggestion
        if (false === $result || (is_array($result) && count($result))) {
```

Is maybe more explicit, wdyt?
Because having a variable named `$errors` saying `false` means, there is no errors :sweat_smile:

Reply:
Yep! Good idea